pom.xml

```diff
@@ -893,6 +899,22 @@
           </configuration>
         </plugin>
 
+        <!-- Ignore runtime-only dependencies in analysis -->
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-dependency-plugin</artifactId>
+          <version>2.10</version>
+          <configuration>
+            <ignoreNonCompile>true</ignoreNonCompile>
+
+            <!-- This dependency is only used for an annotation which doesn't 
+                 show up in bytecode, so doesn't show up as used. -->
+            <ignoredUnusedDeclaredDependencies>
+              <ignoredUnusedDeclaredDependency>com.google.code.findbugs:findbugs-annotations</ignoredUnusedDeclaredDependency>
```

Comment:
The actual fix here is, I think, to exclude jsr305 which presumably was pulled in by bytebuddy. Otherwise everyone downstream has trouble with the dependency analysis as well.